### ISSUE TITLE
如何使用F1保存最佳的分类模型

### ISSUE THREAD

**Issue by CHOUDDIDIBUYAOSONG** [modelscope/ms-swift#4746]
CUDA_VISIBLE_DEVICES=0,1 \
swift sft \
  --model Qwen/Qwen2.5-7B-Instruct \
  --train_type lora \
  --dataset ./data \
  --split_dataset_ratio 0.1 \
  --torch_dtype bfloat16 \
  --num_train_epochs 10 \
  --per_device_train_batch_size 8 \
  --per_device_eval_batch_size 8 \
  --learning_rate 1e-4 \
  --lora_rank 8 \
  --lora_alpha 32 \
  --target_modules all-linear \
  --gradient_accumulation_steps 2 \
  --metric acc \
  --eval_steps 10 \
  --save_steps 20 \
  --save_total_limit 2 \
  --load_best_model_at_end \
  --metric_for_best_model acc \
  --greater_is_better True \
  --logging_steps 10 \
  --max_length 2048 \
  --output_dir output \
  --system ''\
  --warmup_ratio 0.05 \
  --dataloader_num_workers 4 \
  --model_author swift \
  --model_name swift-robot

**Comment by Jintao-Huang** [comment]
你需要自定义metric，并将metric_for_best_model设置为对应metric
